pkg/sdn/plugin/networkpolicy.go

```diff
@@ -83,11 +78,8 @@ func (np *networkPolicyPlugin) Start(node *OsdnNode) error {
 	}
 
 	otx := node.oc.NewTransaction()
-	otx.AddFlow("table=21, priority=200, ip, nw_dst=%s, actions=goto_table:30", np.node.networkInfo.ServiceNetwork.String())
-	otx.AddFlow("table=21, priority=100, ip, actions=ct(zone=%d,commit,table=30)", networkPolicyConntrackZone)
-	otx.AddFlow("table=80, priority=50, ip, actions=ct(zone=%d,commit,table=81)", networkPolicyConntrackZone)
-	otx.AddFlow("table=81, priority=100, ip, ct_state=+trk+est, actions=output:NXM_NX_REG2[]")
-	otx.AddFlow("table=81, priority=0, actions=drop")
+	otx.AddFlow("table=21, priority=200, ip, nw_dst=%s, actions=ct(commit,table=30)", np.node.networkInfo.ClusterNetwork.String())
+	otx.AddFlow("table=80, priority=200, ip, ct_state=+rpl, actions=output:NXM_NX_REG2[]")
```

Comment:
So this basically reverts #13877; we added the ct_zone because the table=21 rule was causing problems with pod-to-external connections, but actually we only need to call commit there on pod-to-pod connections anyway (so they get tracked so we can recognize replied to them as replied). So by only calling commit in a case where iptables will never see the packet anyway, we can avoid needing to fiddle with a zone. The table=80 rule didn't need to be calling commit anyway, it just needed to call ct() on the packet so we can then read its ct_state, but the new rules added in ovscontroller.go end up calling ct() on all packets now, so we don't need to do it again here.